fastlane/lib/fastlane.rb

```diff
@@ -26,7 +26,6 @@ module Fastlane
   Helper = FastlaneCore::Helper # you gotta love Ruby: Helper.* should use the Helper class contained in FastlaneCore
   UI = FastlaneCore::UI
   ROOT = Pathname.new(File.expand_path('../..', __FILE__))
-  DESCRIPTION = "The easiest way to automate beta deployments and releases for your iOS and Android apps"
```

Comment:
Let's leave this in `fastlane.rb`, it makes more sense to be there, as it's not version specific 👍